recipes/thatool/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "thatool" %}
+{% set version = "1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 843304b7768932e6231f3f5ed5f8946b3ec3ebd3d4ac4a75bfba7ea85604739f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```

```suggestion
  number: 0
  noarch: python
  script: "{{ PYTHON }} -m pip install . -vv"
  
```

Let's try to make this a noarch.